.github/workflows/readme_updater.yml

```diff
@@ -43,631 +43,631 @@ jobs:
       - name: Update readme for dockerhub
         run: ./scripts/update_readme_dh.sh
       - name: Docker Hub Description Apache Airflow
-        uses: peter-evans/dockerhub-description@v4
+        uses: peter-evans/dockerhub-description@v3
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
           repository: rapidfort/airflow
           readme-filepath: ./community_images/airflow/airflow/bitnami/README.md.dockerhub
           short-description: RapidFort optimized, hardened image for Apache Airflow
       - name: Docker Hub Description Apache Airflow Ironbank
-        uses: peter-evans/dockerhub-description@v4
+        uses: peter-evans/dockerhub-description@v3
```

Comment:
why have we downgraded from v4 to v3?

Reply:
This is some bug in execution path of `image_generator.sh`. It lowers this version. But some bot raises PR which is changing it back to v4. Ill see to it and fix it if possible.